notebooks/istio_example.ipynb

```diff
@@ -29,7 +29,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Error from server (AlreadyExists): namespaces \"seldon\" already exists\r\n"
```

Comment:
Do we need this notebook change added or is it an artifact of your testing?

Reply:
`"seldonio/mock_classifier:1.7.0-dev"` is now returned in the Rest and gRPC Response objects: i.e.  `meta.requestPath.model` is now present where previously `meta` was an empty object. When glancing the docs, it might be useful to see that this metadata exists.

Existing notebook with empty metadata:

<img width="1192" alt="image" src="https://user-images.githubusercontent.com/5828176/110774396-a828a880-8255-11eb-9a71-5fef8fbd325a.png">


New notebook with Response object metadata filled:

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/5828176/110774294-8deeca80-8255-11eb-8bd8-48061c02db97.png">



Your call though. The notebook doesn't necessarily need to be changed: the changes here update the minor existing error messages in the cell outputs – the cell inputs haven't been changed. Although they're minor, if you're skimming the docs for the first time and see errors, you might be concerned.

The minor existing error messages are just related to the idempotency of `kubectl create` (i.e. attempting to create an existing resource):

<img width="1192" alt="image" src="https://user-images.githubusercontent.com/5828176/110772572-a958d600-8253-11eb-9ab4-28e0e60073de.png">

<img width="1176" alt="image" src="https://user-images.githubusercontent.com/5828176/110772658-c2618700-8253-11eb-9ae5-0445ae07a189.png">